site/pages/component/gc-table-en.hbs

```diff
@@ -0,0 +1,180 @@
+---
+{
+	"altLangPrefix": "gc-table",
+	"breadcrumb": [
+		{ "title": "GCWeb home", "link": "https://wet-boew.github.io/themes-dist/GCWeb/index-en.html" },
+		{ "title": "Provisional functionality", "link": "https://wet-boew.github.io/themes-dist/GCWeb/provisional-en.html" }
+	],
+	"dateModified": "2020-10-16",
+	"description": "Working examples for tables - provisional feature",
+	"language": "en",
+	"parentdir": "component",
+	"tag": "table",
+	"title": "Tables - Provisional feature"
+}
+---
+{{>alert-provisional-demos}}
+<span class="wb-prettify all-pre hide"></span>
+
+<dl class="horizontal">
+	<dt>Status</dt>
+	<dd>Provisional</dd>
+	<dt>Type</dt>
+	<dd>Canada.ca design pattern</dd>
+	<dt>Last review</dt>
+	<dd>2021-02-01</dd>
+	<dt>Conforming to</dt>
+	<dd>Content and IA spec 2.1</dd>
+	<dt>Guidance</dt>
+	<dd><a href="#">Tables - Canada.ca design pattern [coming soon]</a></dd>
+	<dt>Technical name</dt>
+	<dd>
+		<ul>
+			<li><code>gc-table</code></li>
+		</ul>
+	</dd>
+</dl>
+
+<h2>Working example</h2>
+
+<p>Responsive cards</p>
+<table class="provisional gc-table table" id="myTable">
+		<caption class="wb-inv">Population growth in Canadian cities</caption>
```

Comment:
One or the other
* remove the caption element and ensure the previous paragraph/heading explain the table context
* Display the caption element